packages/elements/src/components/ino-markdown-editor/ino-markdown-editor.tsx

```diff
@@ -247,6 +268,13 @@ export class MarkdownEditor implements ComponentInterface {
               <ino-icon icon="code_block"/>
             </button>
           </div>
+          <ino-popover
+            placement="top-start"
+            color-scheme="transparent"
+            controlled={true}
+            visible={!this.errorMessage == false}>
```

Comment:
Does `!this.errorMessage` not suffice?

Reply:
No, because I need the double inverted value 😅 
As alternative I can use `this.errorMessage !== ''` Then I ignore `null` or `undefinded` as invalid values.

![image](https://user-images.githubusercontent.com/23154336/150341762-26836b4b-1d1c-4a53-bdbc-e719725fd5b2.png)

Reply:
And what about `Boolean(this.errorMessage)`?

![image](https://user-images.githubusercontent.com/22963121/150499967-4a075c37-ba51-404e-a403-0775908e36df.png)

Reply:
Crazy, new insights 👍 😃